standard/expressions.md

```diff
@@ -1456,6 +1458,11 @@ A *simple_name* is either of the form `I` or of the form `I<A₁, ..., Aₑ>`,
     - Otherwise, if the namespaces imported by the *using_namespace_directive*s of the namespace declaration contain exactly one type having name `I` and `e` type parameters, then the *simple_name* refers to that type constructed with the given type arguments.
     - Otherwise, if the namespaces imported by the *using_namespace_directive*s of the namespace declaration contain more than one type having name `I` and `e` type parameters, then the *simple_name* is ambiguous and a compile-time error occurs.  
   > *Note*: This entire step is exactly parallel to the corresponding step in the processing of a *namespace_or_type_name* ([§7.8](basic-concepts.md#78-namespace-and-type-names)). *end note*
+- Otherwise, if `e` is zero and `I` is the identifier `_`, the *simple_name* is a discard () and its type shall be inferred from the syntactic context:
```

Comment:
See my comment at the end of variables. I Think this wording disallows multiple discards in the same scope.

Reply:
Can you elaborate what gives you that impression? I feel like this wording very explicitly gives each discard its own type.

For now I'm adding the word "fresh" ("the *simple_name* is a fresh discard") and a section reference to Discards in variables.md, and mark this resolved, but please reopen if that doesn't address your concern.